manifests/05_clusterrole.yaml

```diff
@@ -271,7 +271,8 @@ rules:
 - apiGroups:
   - sharedresource.openshift.io
   resources:
-    - shares
+    - sharedConfigs
```

Comment:
notice up in line 272 here .... with all the various rename churn, we missed changing this to what is now the "old" value, `storage.openshift.io`

we go lucky, in that we wanted on `sharedresource.openshift.io`